celery/utils/time.py

```diff
@@ -190,9 +190,6 @@ def remaining(start, ends_in, now=None, relative=False):
         ~datetime.timedelta: Remaining time.
     """
     now = now or datetime.utcnow()
-    if str(start.tzinfo) == str(now.tzinfo) and now.utcoffset() != start.utcoffset():
```

Comment:
since we are doing to remove this part, wouldn't it better to have some test to make sure this change won't create any regression?

Reply:
all the unit tests are failing

Reply:
Looking into them

Reply:
Added unit tests.